native/packages/react-native-bpk-theming/package-lock.json

```diff
@@ -143,15 +203,66 @@
 				"prop-types": "15.6.0"
 			}
 		},
+		"react-native-bpk-component-button": {
+			"version": "3.1.0",
+			"resolved": "https://registry.npmjs.org/react-native-bpk-component-button/-/react-native-bpk-component-button-3.1.0.tgz",
+			"integrity": "sha512-YjPwPpniYDEq11e9LyNR1rS6VhJKn7bVuuGv5+hr5+DbUWI2MedQfF/QvxnMl/hn8zRz0PL/aj3Zc5PPlcORYQ==",
+			"dev": true,
+			"requires": {
+				"bpk-tokens": "26.4.0",
+				"lodash": "4.17.4",
+				"prop-types": "15.6.0",
+				"react-native-bpk-component-text": "2.1.10",
+				"react-native-bpk-theming": "1.0.3",
+				"react-native-linear-gradient": "2.3.0"
+			}
+		},
+		"react-native-bpk-component-text": {
+			"version": "2.1.10",
+			"resolved": "https://registry.npmjs.org/react-native-bpk-component-text/-/react-native-bpk-component-text-2.1.10.tgz",
+			"integrity": "sha512-SptaBAS/cZwyyTUrjrQeJlW7UPBCRfILgS6bAXYKrmJgvED3TlHQ0rtYONF51d2HZUcMEPKnwCjB6GLDSDldbg==",
+			"dev": true,
+			"requires": {
+				"bpk-tokens": "26.4.0",
+				"prop-types": "15.6.0"
+			}
+		},
+		"react-native-bpk-theming": {
+			"version": "1.0.3",
+			"resolved": "https://registry.npmjs.org/react-native-bpk-theming/-/react-native-bpk-theming-1.0.3.tgz",
+			"integrity": "sha512-/raKBcJYtE+Ty0G+GpMbPZAoCGNvm4McQ0sFXZ/+QwF/f7u02m5dZWA3AKPjOiHWE5fZZya3rQGTFr/Af77Taw==",
+			"dev": true,
+			"requires": {
+				"theming": "1.1.0"
+			}
+		},
+		"react-native-linear-gradient": {
+			"version": "2.3.0",
+			"resolved": "https://registry.npmjs.org/react-native-linear-gradient/-/react-native-linear-gradient-2.3.0.tgz",
+			"integrity": "sha512-34z/F/0XxqF/9Jt8zsWENiYehj8xywRpZOfUDfanshSSAZkStLHThScyagvOAzkkCxrUV1K5sWS5Hxu6iR6VTA==",
+			"dev": true,
+			"requires": {
+				"prop-types": "15.6.0"
+			}
+		},
 		"setimmediate": {
 			"version": "1.0.5",
 			"resolved": "https://registry.npmjs.org/setimmediate/-/setimmediate-1.0.5.tgz",
 			"integrity": "sha1-KQy7Iy4waULX1+qbg3Mqt4VvgoU="
 		},
+		"simple-swizzle": {
+			"version": "0.2.2",
+			"resolved": "https://registry.npmjs.org/simple-swizzle/-/simple-swizzle-0.2.2.tgz",
+			"integrity": "sha1-pNprY1/8zMoz9w0Xy5JZLeleVXo=",
+			"dev": true,
+			"requires": {
+				"is-arrayish": "0.3.1"
+			}
+		},
 		"theming": {
 			"version": "1.1.0",
 			"resolved": "https://registry.npmjs.org/theming/-/theming-1.1.0.tgz",
-			"integrity": "sha512-YRhQV+eqeuef70eqfMGKGR+3DBXoWJ4lsrSSuj8dzrv0/EsnVuDmZAjIifESvJARtuP1KbMpdl8qbXdLAz4t/w==",
+			"integrity": "sha1-BWJ2C1WhuRnC1e65QTA1H4lY4To=",
```

Comment:
why are these sha1's whereas most of them are sha512? 🤔

Reply:
Honestly no idea, something in my NPM install seems to keep doing this though. I'll see if I can find anything.

Reply:
Looks like it's a known issue https://github.com/npm/npm/issues/17749. Not that it helps us to know that!